frontend/src/shared/components/register-page/register-page.component.html

```diff
@@ -0,0 +1,88 @@
+                        <span class="error" *ngIf="emailControl.errors?.['email']">Incorrect email format</span>
+                    </div>
+                    <div class="input-container">
+                        <div class="label-container full-width">
+                            <label>Password</label>
+                        </div>                        
+                        <div class="password-container">
+                            <input class="input full-width" [ngClass]="{
+                                'error': (passwordControl.invalid && (passwordControl.dirty || passwordControl.touched))
+                                }"
+                                [type]="hidePass ? 'password' : 'text'" 
+                                placeholder="Write password"
+                                [value]="password"
+                                [(ngModel)]="password"
+                                (ngModelChange)="resetPasswordControl()"
+                                formControlName="passwordControl"
+                              />
+                              <mat-icon class="hide-icon" (click)="hidePass = !hidePass">{{hidePass ? 'visibility' : 'visibility_off'}}</mat-icon>
+                        </div>                        
+                        <span class="error" *ngIf="passwordControl.errors?.['required'] && (passwordControl.dirty || passwordControl.touched)">Password is required.</span>
+                        <span class="error" *ngIf="passwordControl.errors?.['minlength']">Must be at least 8 characters</span>
+                    </div>
+                    <div class="input-container">
+                        <div class="label-container full-width">
+                            <label>Repeat password</label>
+                        </div>                        
+                        <div class="password-repeat-container">
+                            <input class="input full-width" [ngClass]="{
+                                'error': (passwordRepeatControl.invalid && (passwordRepeatControl.dirty || passwordRepeatControl.touched))
+                                }"
+                                [(ngModel)]="passwordRepeat"
+                                [type]="hidePassRepeat ? 'password' : 'text'" 
+                                placeholder="Repeat password"
+                                formControlName="passwordRepeatControl"
+                              />
+                              <mat-icon class="hide-icon" (click)="hidePassRepeat = !hidePassRepeat">{{hidePassRepeat ? 'visibilityRepeat' : 'visibility_off'}}</mat-icon>
+                        </div>                        
+                        <span class="error" *ngIf="passwordRepeatControl.errors?.['required'] && (passwordRepeatControl.dirty || passwordRepeatControl.touched)">Password repeat is required.</span>
```

Comment:
It's a redundancy, `Passwords do not match` is enough

Reply:
Ok